package.json

```diff
@@ -9,9 +9,9 @@
     "start": "cd docs && yarn run develop",
     "dist": "concurrently yarn:dist:*",
     "dist:rollup": "rm -rf dist && cross-env NODE_ENV=production rollup -c",
-    "dist:transpile:cjs": "rm -rf lib && cross-env BABEL_MODULE=commonjs babel src --out-dir lib --extensions '.js,.ts,.jsx,.tsx'",
-    "dist:transpile:esm": "rm -rf lib-esm && babel src --out-dir lib-esm --extensions '.js,.ts,.jsx,.tsx'",
-    "dist:types": "rm -rf types && tsc",
+    "dist:transpile:cjs": "rm -rf lib && tsc -p tsconfig.json --outDir lib --module commonjs",
```

Comment:
1. Do we need to continue to publish transpiled version of PRC, or can we publish the TS alone?

2. TODO: Review the transpiled output for any obvious issues.